cloudprem/main.tf

```diff
@@ -245,6 +245,26 @@ module "memcached" {
   cluster_size  = 1
   instance_type = var.cache_instance_type
 }
+module "webhooks" {
+  source = "./modules/webhooks"
+
+  depends_on = [module.eks_cluster]
+
+  name = local.identifier
+
+  vpc_id     = local.vpc_id
+  subnet_ids = local.private_subnet_ids
+
+  allowed_cidr_blocks = [local.vpc_cidr]
+
+  cluster_size  = local.azs_count
+  rds_kms_key_id = var.rds_kms_key_id
+
+  rds_address = module.primary_database.this_db_instance_address
+  rds_user = module.primary_database.this_db_instance_username
+  rds_pass = random_password.primary_database.result
```

Comment:
We're using the "primary" RDS instance for webhooks as well? The description mentioned giving it it's own DB/RDS? Might be outdated...

Reply:
I didn't see a reason to spin up an additional RDS database just for this so we create the frontegg schemas on the main DB. Open to suggestions if that's problematic.

Reply:
No, that's fine. It just makes the PR description a little misleading..

I dont have any problem with frontegg using the main RDS... Might be something to keep an eye on though performance-wise?